Subject: [PATCH] QS Tile: increase DEFAULT_MAX_BOUND to 6

With more and more apps adding tiles increase
the default max bound to 6 to account for
the increase.

diff --git a/packages/SystemUI/src/com/android/systemui/qs/external/TileServices.java b/packages/SystemUI/src/com/android/systemui/qs/external/TileServices.java
index 6bc94b2..b972f512 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/external/TileServices.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/external/TileServices.java
@@ -49,7 +49,7 @@
  * Runs the day-to-day operations of which tiles should be bound and when.
  */
 public class TileServices extends IQSService.Stub {
-    static final int DEFAULT_MAX_BOUND = 3;
+    static final int DEFAULT_MAX_BOUND = 6;
     static final int REDUCED_MAX_BOUND = 1;
 
     private final ArrayMap<CustomTile, TileServiceManager> mServices = new ArrayMap<>();
